request.go

```diff
@@ -68,7 +70,7 @@ func (conn *Connection) SelectMany(space, index interface{}, offset, limit, iter
 		s := reflect.ValueOf(keys)
 
 		for i := 0; i < s.Len(); i++ {
-			fut := conn.SelectAsync(space, index, offset, limit, iterator, s.Index(i).Interface())
+			fut := conn.SelectAsync(space, index, 0, 4294967295, iterator, s.Index(i).Interface())
```

Comment:
Use `offset+limit` instead of `4294967295`. It will safe at least performance of first page.

Reply:
done